### ISSUE TITLE
Autofixing?

### ISSUE THREAD

**Issue by merlinstardust** [not-an-aardvark/eslint-rule-composer#7]
Is there a means to add autofixing to composed rules?

If so, how would I go about doing that.

Specifically I'm looking to autofix no-unused-vars as reported [here](https://github.com/eslint/eslint/issues/11668)

**Comment by sweepline** [comment]
There is already a package that just does that. Take a look at [eslint-plugin](https://github.com/aladdin-add/eslint-plugin/blob/master/packages/autofix/lib/rules/no-unused-vars.js)

**Comment by merlinstardust** [comment]
That was added after my original question. I would still like to know how I go about doing that so I can learn how to do so for myself

**Comment by sweepline** [comment]
Yes, so if you go to my link, and read the code you will se a solution to an autofixer for `no-unused-vars` written using `eslint-rule-composer` which I presume was your original question?

**Comment by merlinstardust** [comment]
The code technically answers the question but it doesn't walk me through it. I'm hoping for a brief tutorial or blog post on how this works. Reading code is not a substitute for teaching
